lib/app/widgets/app_bar.dart

```diff
@@ -114,4 +117,81 @@ class _RefreshButtonState extends State<RefreshButton> {
   void _flushCachedData() {
     campaignActionCache.flushCache();
   }
+
+  void _storeCacheDataOnDevice() async {
+    final theme = Theme.of(context);
+    final shouldStoreCache = await showDialog<bool>(
+      context: context,
+      barrierDismissible: true,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          backgroundColor: ThemeColors.alertBackground,
+          content: Text(
+            t.campaigns.debug.confirm_storing_cache,
+            textAlign: TextAlign.center,
+            style: theme.textTheme.labelMedium?.apply(
+              color: theme.colorScheme.surface,
+              fontSizeDelta: 1,
+            ),
+          ),
+          actionsAlignment: MainAxisAlignment.spaceBetween,
+          actions: [
+            TextButton(
+              onPressed: () => Navigator.maybePop(context, false),
+              child: Text(
+                t.common.actions.cancel,
+                style: theme.textTheme.labelLarge?.apply(color: ThemeColors.textCancel),
+              ),
+            ),
+            TextButton(
+              onPressed: () => Navigator.maybePop(context, true),
+              child: Text(
+                t.common.actions.save,
+                style: theme.textTheme.labelLarge?.apply(
+                  color: ThemeColors.textWarning,
+                  fontWeightDelta: 2,
+                ),
+              ),
+            ),
+          ],
+        );
+      },
+    );
+    if (shouldStoreCache ?? false) {
```

Comment:
Is the `?? false` a leftover?

Reply:
no. it could be null. So the fallback is false.